setup.py

```diff
@@ -869,7 +869,8 @@ def detect_readline_curses(self):
             curses_library = readline_termcap_library
         elif self.compiler.find_library_file(self.lib_dirs, 'ncursesw'):
             curses_library = 'ncursesw'
-        elif self.compiler.find_library_file(self.lib_dirs, 'ncurses'):
+        # Issue 36210: AIX and ncurses does not co-exist with IBM supplied curses
+        elif not HOST_PLATFORM.startswith("aix") and self.compiler.find_library_file(self.lib_dirs, 'ncurses'):
```

Comment:
Please respect [PEP 8](https://www.python.org/dev/peps/pep-0008/) when adding new code. This line is too long.

Reply:
nods. corrected.